advocacy_docs/partner_docs/VeritasGuide/04-ConfiguringTheIntegration.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Configuring the Integration'
+description: 'Walkthrough on configuring NetBackup for PostgreSQL'
+---
+
+
+### Prerequisites
+
+- A running EDB Postgres Advanced (EPAS)
+- A running NetBackup Server with a DataStore Policy set up
+- NetBackup Client installed
+- NetBackup PostgreSQL Agent installed
+
+
+### Configuring (Veritas NetBackup for PostgreSQL)
+
+The following steps show an example of how to configure NetBackup for PostgreSQL for EDB Postgres Advanced 13 on CentOS 7.
+
+!!! Note
+    For detailed information, refer to the [Veritas NetBackup for PostgreSQL Administrator's Guide](https://www.veritas.com/content/support/en_US/doc/129277259-150015228-0/v129277275-150015228).
```

Comment:
don't need () around text.

Reply:
This is for markdown formatting so the url does not up as text only as a hyperlink in the published guide. You can verify that by viewing the deployed mockup